documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -701,6 +701,10 @@ The placeholder, `${topic}`, is replaced by the topic name.
 |Specifies the schema name where the PostgreSQL PostGIS extension is installed.
 The default is `public`; however, if the PostGIS extension was installed in another schema, this property should be used to specify the alternate schema name.
 
+|[[jdbc-property-dialect-sqlserver-identity-insert]]<<jdbc-property-dialect-sqlserver-identity-insert, `+dialect.sqlserver.identity.insert+`>>
+|`false`
+|Specifies whether the connector automatically enables identity inserts before an _insert_ or _upsert_ operation and is disabled immediately afterward.
```

Comment:
```suggestion
|Specifies whether the connector automatically sets an `IDENTITY_INSERT` before an `INSERT` or `UPSERT` operation into the identity column of SQL Server tables , and then unsets it immediately after the operation.
```

Reply:
The convention in the SQL Server documentation seems to be to capitalize SQL commands, so I followed that model.